.travis.yml

```diff
@@ -28,6 +28,24 @@ jobs:
       # Exclude the default Python 3.5 build
       - python: 3.5
   include:
+    - stage: Build and test wheel
+      os: linux
+      arch: arm64
```

Comment:
@janaknat mentioned in #92 that `arm64-graviton2` may produce faster builds?

Reply:
Stages run sequentially and if any of the stage is failing, the remaining stages will not run.

Reply:
I can see the scipy-wheel builds are running on travis-ci.org and arm64-graviton2 is available only on travis-ci.com. Please have a look at the discussion [here](https://github.com/pyca/cryptography/issues/5292.) Using arm64-graviton2 will require scipy-wheel builds to migrate on travis-ci.com. Also, the method/configuration to use arm64-graviton2 is not officially declared.

Reply:
Travis.com offers free usage to public/OSS repositories, and the migration should be simple as long as this project does not have any weird dependencies.  Its highly recommended to move sooner rather than later as .com is the only platform that will offer arm64 builds on graviton2 instances.